_rules/element-lang-valid-de46e4.md

```diff
@@ -97,37 +104,73 @@ The `lang` attribute value has a valid language tag, but a syntactically invalid
 
 #### Failed Example 1
 
-The `lang` attribute value is not a primary language subtag.
+The `lang` [attribute value][] is not a primary language subtag.
 
 ```html
 <html>
 	<body>
-		<article lang="dutch"></article>
+		<article lang="dutch">
+			Zij liepen een vreemde Tiki bar binnen, aan de rand van een dorpje aan het strand.
+		</article>
 	</body>
 </html>
 ```
 
 #### Failed Example 2
 
-The `lang` attribute value is not empty ("") and is not a primary language subtag.
+The `lang` [attribute value][] is not empty (`""`) and is not a primary language subtag.
 
 ```html
 <html>
 	<body>
-		<article lang="#!"></article>
+		<article lang="#!">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
 	</body>
 </html>
 ```
 
 #### Failed Example 3
 
-The `lang` attribute value consists of only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace) and is not a [valid language tag][].
+The `lang` [attribute value][] consists of only [whitespace][] and is not a [valid language tag][].
 
 ```html
 <html>
 	<body>
-		<article lang=" ">
-			The quick brown fox jumped over the lazy dog
+		<article lang="  ">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+The `lang` [attribute value][] has a valid language tag, but its [descendant][] is not [included in the accessibility tree][].
+
+```html
+<html>
+	<body>
+		<article lang="english">
+			<p aria-hidden="true">
+				They wandered into a strange Tiki bar on the edge of the small beach town.
+			</p>
+		</article>
+	</body>
+</html>
+```
+
+#### Failed Example 5
+
+The `lang` [attribute value][] has a valid language tag, but its [descendant][] is not [visible][] though it is still [included in the accessibility tree][].
```

Comment:
This is also inapplicable, right?

Reply:
See - https://github.com/act-rules/act-rules.github.io/pull/1383#discussion_r501678774